docs/index.rst

```diff
@@ -16,10 +16,24 @@ can focus on the content instead.
 
 Try it out now using the `Black Playground <https://black.now.sh>`_.
 
-.. note::
-
-   `Black is beta <installation_and_usage.html#note-this-is-a-beta-product>`_.
+.. admonition:: Note - this is a beta product
+
+   *Black* is already `successfully used <https://github.com/psf/black#used-by>`_ by
+   many projects, small and big. It also sports a decent test suite. However, it is
+   still very new. Things will probably be wonky for a while. This is made explicit by
+   the "Beta" trove classifier, as well as by the "b" in the version number. What this
+   means for you is that **until the formatter becomes stable, you should expect some
+   formatting to change in the future**. That being said, no drastic stylistic changes
```

Comment:
Not to fussed here, take it of leave it. I just don't like a ~3 year old project being called new.
```suggestion
   many projects, small and big. Black has a comprehensive test suite, with efficient parallel tests, our own auto  formatting and parallel Continuous Integration runner. However, _Black_ is still beta. Things will probably be wonky for a while. This is made explicit by
   the "Beta" trove classifier, as well as by the "b" in the version number. What this
   means for you is that **until the formatter becomes stable, you should expect some
   formatting to change in the future**. That being said, no drastic stylistic changes
```